lib/tasks/fix_maine_nil_counties.rake

```diff
@@ -35,14 +42,15 @@ namespace :migrations do
     pull_county
 
     def address_needs_fixing?(address)
-      address.county.blank? && !address.zip.blank?
+      address.county.blank? && !address.zip.blank? || address.county.downcase.include?("benefitmarket")
```

Comment:
are we adding this line simply to make sure that we haven't saved the object to the address object?

Reply:
No this identifying ones that already had the object saved to the address since it would have the text "benefitmarket" in it.